GVRf/Framework/framework/src/main/res/raw/yuv_nv21_to_rgb.fsh

```diff
@@ -0,0 +1,25 @@
+precision highp float;
+in vec2 diffuse_coord;
+
+#ifdef VULKAN
+layout (set = 0, binding = 4)
+#endif
+uniform sampler2D y_texture;
+
+#ifdef VULKAN
+layout (set = 0, binding = 5)
+#endif
+uniform sampler2D uv_texture;
+
+out vec4 fragColor;
```

Comment:
This idea does not work because native code will blow up if it sees 'layout' without 'in', 'out', or 'uniform' on the line. TODO: fix shader munging (Native code) and layout specs (shader source) at the same time.